docs/en/setup/backend/logs-template.json

```diff
@@ -0,0 +1,82 @@
+{
+    "timestamp": {
+        "type": "sequence",
+        "min": "1649643929000",
+        "max": "1649653929000"
+    },
+    "serviceName": {
+        "type": "randomString",
+        "length": "20",
+        "prefix": "test_svc_name_",
+        "letters": true,
+        "numbers": true
+    },
+    "serviceInstanceName": {
+        "type": "randomString",
+        "length": "20",
+        "prefix": "test_svc_inst_name_",
+        "letters": true,
+        "numbers": true
+    },
+    "endpointName": {
+        "type": "randomString",
```

Comment:
For `randomString` using in service, instance and endpoint, we should have the number of candidates.

Reply:
> For `randomString` using in service, instance and endpoint, we should have the number of candidates.

You can just specify `domainSize` to limit the count of candidates

Reply:
FYI @hanahmily Please notice this when you use this tool